src/DurableTask.Core/Query/OrchestrationQueryResult.cs

```diff
@@ -11,25 +11,35 @@
 //  limitations under the License.
 //  ----------------------------------------------------------------------------------
 
-using System.Collections.Generic;
-
 namespace DurableTask.Core.Query
 {
+    using System.Collections.Generic;
+
     /// <summary>
     /// The status of all orchestration instances with paging for a given query.
     /// </summary>
     public class OrchestrationQueryResult
     {
         /// <summary>
-        /// Gets or sets a collection of statuses of orchestration instances matching the query description.
+        /// Constructor of OrchestrationQueryResult Class.
+        /// </summary>
+        /// <param name="OrchestrationState">A collection of orchestration instance status values.</param>
+        /// <param name="ContinuationToken">A server-generated continuation token or <c>null</c> if there are no further continuations.</param>
+        public OrchestrationQueryResult(IReadOnlyCollection<OrchestrationState> OrchestrationState, string ContinuationToken)
```

Comment:
Parameter names must start with a lowercase letter.
```suggestion
        public OrchestrationQueryResult(IReadOnlyCollection<OrchestrationState> orchestrationState, string continuationToken)
```